packages/usa-form/src/usa-form.stories.js

```diff
@@ -10,6 +10,12 @@ import EsContent from "../../templates/usa-sign-in/usa-sign-in~lang-es.json";
 import MultipleContent from "../../templates/usa-sign-in/usa-sign-in--multiple/usa-sign-in--multiple.json";
 import EsMultipleContent from "../../templates/usa-sign-in/usa-sign-in--multiple/usa-sign-in--multiple~lang-es.json";
 
+const InfoContent = {
+  modifier: "usa-alert--warning",
+  title: "Heads up!",
+  text: "This is a developer preview. The patterns presented on this page may not be suitable production level code.",
+};
+
```

Comment:
**polish:** This isn't part of the form story. You could move this to where the template is being called. Example below.

https://github.com/uswds/uswds/blob/42907b9ef1a8dfa859730e91d03482f2283b0392/packages/templates/usa-password-reset-form.twig#L20-L30

Reply:
WOW this is an awesome change! Being able to isolate all the alert data right within the template is a great improvement. Thanks for flagging this!

Resolved in [b65075a](https://github.com/uswds/uswds/pull/6029/commits/b65075a91e6030998612cd38b4c5c42bfc1c666d)